### PR TITLE
Dmirano/3493 mmis activities dashboard

### LINKED ISSUE #3493: [Feature] Update Activities Dashboard
**Description and related issues -**

**For MMIS add an example box that says**

"Examples of individual activities include but are not limited to: E&E online application, E&E worker portal, E&E noticing system, claims processing engine, claims processing prior authorization, claims processing record keeping, HIE interface(s), CQM databases, decision support systems analytics platform, provider enrollment screening, provider fraud/waste/abuse, managed care member enrollment, etc. For each activity added, you will have the opportunity to provide details in their own respective subsections."

**For MMIS remove funding source name from Activity List**

** Display error message for no activities when admin check on**

If there are no activities and admin check is on, then this message should be displayed above the Add activity button, should have been handled in #3349

Figma: https://www.figma.com/file/hJpKHKU6fz5J0Z7fisSwa2/eAPD-MMIS-2022?node-id=41:5864&t=GKsE3Y4JOR7FbJTU-0

### This task is done when…
- [ ] for MMIS add example text
- [ ] for MMIS remove funding source from activity name
- [ ] show empty activity message when admin check is on
- [ ] empty activity messages shows in Admin Check panel
- [ ] clicking the empty activity message in the Admin Check panel takes you to the Activities Dashboard

### PR DESCRIPTION
**Pull request by mirano-darren**
Resolves #3493 

### Description
Updates the activities dashboard for MMIS APDs

### Significant changes or possible side effects
Also adds unit tests for the SecurityPlanning page

### Automated test cases written

| Given | When | Then | Type (jest, tap, cypress) |
| ----- | ---- | ---- | ------------------------- |
| A MMIS APD | On the activities dashboard | Help box is added, funding types are removed from activity names | jest | 
| A MMIS APD | On the security planning page | page loads existing data | jest | 

### Steps to manually verify this change

1. Code has been reviewed  

### This pull request is ready to code review when

- [x] Automated tests are updated (and all tests are passing)
- [x] New automated test cases are documented above
- [x] Pull request has been labeled, if applicable with feature, content, bug,
      tests, refactor
- [ ] Associated OpenAPI documentation has been updated
- [x] The experience passes a basic manual accessibility audit (keyboard nav,
      screenreader, text scaling) OR an exemption is documented

### This pull request is ready to test when

- [ ] Code has been reviewed by someone other than the original author

### This pull request is ready to review when the QA has

- [ ] Verified the functionality related to the change
- [ ] Verified that the change works with Narrator on Windows
- [ ] Verified that the change works with VoiceOver on Mac
- [ ] Verified all updated pages with the WAVE tool
- [ ] Verified tab and keyboard navigation functionality

### This pull request can be merged when

- [ ] Design has approved the experience
- [ ] Product has approved the experience
- [ ] - [ ]

### CODE REVIEW

File: web/src/i18n/locales/en/securityPlanning.yaml [hunk 1]
```diff
@@ -12,7 +12,7 @@ securityInterfacePlan:
 
 bcDrplan:
   instruction:
-    heading: Business Continuity and Disaster Recovery Plan
+    label: Business Continuity and Disaaster Recovery Plan
```
Comment (amyd11) [inline web/src/i18n/locales/en/securityPlanning.yaml:15]: **Typo:** Diaster's got an extra "a" in it

**Review by amyd11 — APPROVED** [review]
Only thing to fix is that small typo, otherwise everything looks good!

**Review by thetif — APPROVED** [review]

**Review by tbolt — APPROVED** [review]

**Review by jeromeleecms — APPROVED** [review]
Matches the acceptance criteria. Looking at tickets, the export view will be managed by a separate ticket.

**Review by SGilliamA1M — APPROVED** [review]
Changes look good.

**Pull request merged**